packages/terra-demographics-banner/jest/jest-transform-cache--Users-jl021230-Documents-PopulationHealth-terra-terra-core-packages-terra-demographics-banner-45d11b25ca1731bb00daaa0f4666f685/0e/DemographicsBannerDisplay_0eea0e4c2ad7fc5977abdd7d30346168

```diff
@@ -0,0 +1,169 @@
+/* istanbul ignore next */'use strict';var _createClass = function () {function defineProperties(target, props) {for (var i = 0; i < props.length; i++) {var descriptor = props[i];descriptor.enumerable = descriptor.enumerable || false;descriptor.configurable = true;if ("value" in descriptor) descriptor.writable = true;Object.defineProperty(target, descriptor.key, descriptor);}}return function (Constructor, protoProps, staticProps) {if (protoProps) defineProperties(Constructor.prototype, protoProps);if (staticProps) defineProperties(Constructor, staticProps);return Constructor;};}();var /* istanbul ignore next */_react = require('react'); /* istanbul ignore next */var _react2 = _interopRequireDefault(_react);
```

Comment:
Looks like I had a few cached jest tests that got pulled into the pull request. I removed them all here. https://github.com/cerner/terra-core/pull/191/commits/2aeae872d5e6f1851cdb0c14cca7c9706349bb2a